website/docs/installation/ferretdb/rpm.md

```diff
@@ -7,7 +7,13 @@ sidebar_position: 3
 To install the `.rpm` packages for FerretDB on your RHEL, CentOS, and other `.rpm`-based systems,
 you can use `rpm` tool.
 
-Download the latest FerretDB `.rpm` package from [our release pages](https://github.com/FerretDB/FerretDB/releases/latest),
+We provide different `.rpm` packages for various deployments on [our release page](https://github.com/FerretDB/FerretDB/releases/latest)
```

Comment:
Note that the URL is different from other pages. Also – add dot at the end of sentence (or `:` everywhere)